script/Ciel/Day 12/12_01_CIEL12_2B.txt

```diff
@@ -367,7 +367,9 @@ Senpai managed to beat me to the stairs somehow.
 [sha:032e1f80685fcb8c06d8734dfdaf90a9ce37def5]{
 -- Page 53, Offset 31000.
 -- いつも待ってるばかりだから、たまにはこっちから行こうと思ったんです」
-You're always the one doing the visiting, so I figured I'd stop by your classroom for a change."
+// Weird one needs another look
+I feel like I always have to wait too long to see you again,
```

Comment:
slightly altered meaning
> I feel like I'm always the one waiting for you to come to me, so I figured I'd take the initiative once in a while.

Reply:
Yeah that feels like more what he's actually saying to me, but I forget the reason we didn't go with something like that. Maybe Petrikow remembers.